dattri/benchmark/load.py

```diff
@@ -0,0 +1,268 @@
+"""This module provides functions to load predefined benchmark settings."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Dict, Tuple
+
+import os
+import pathlib
+import zipfile
+from functools import partial
+from io import BytesIO
+
+import requests
+import torch
+
+from dattri.benchmark.datasets.mnist import (
```

Comment:
So this is just an example using MNIST or we now only have the stored ckpts/gts for MNIST?

Reply:
Will support more in our huggingface repo (not in this PR)